### PR TITLE
update compression exclude example

### PR DESCRIPTION
**Pull request by J-shang**

### CODE REVIEW

File: dependencies/required.txt [hunk 1]
```diff
@@ -21,3 +21,5 @@ scipy < 1.6 ; python_version < "3.7"
 scipy ; python_version >= "3.7"
 pandas < 1.2 ; python_version < "3.7"
 matplotlib < 3.4 ; python_version < "3.7"
+tornado
```
Comment (ultmaster) [inline dependencies/required.txt:24]: Why do we need tornado as required dep?

File: dependencies/required.txt [hunk 2]
```diff
@@ -21,3 +21,5 @@ scipy < 1.6 ; python_version < "3.7"
...
+jupyter_core
```
Comment (ultmaster) [inline dependencies/required.txt:25]: Why is jupyter required?

File: dependencies/required.txt (hunk 1 above)
Comment (J-shang) [reply dependencies/required.txt:24]: because when I pip install nni in a new env, these two packages are required.

Comment (J-shang) [reply dependencies/required.txt:24]: image

Comment (J-shang) [reply dependencies/required.txt]: reset the requirement

**Review by QuanluZhang — APPROVED** [review]

**Review by ultmaster — APPROVED** [review]

**Pull request merged**